packages/sdk/test/job.js

```diff
@@ -0,0 +1,41 @@
+const path = require('path');
```

Comment:
Write our test in TypeScript, please. See https://github.com/alibaba/pipcook/blob/master/packages/core/src/constants/model_test.ts.